tests/ssl-opt.sh

```diff
@@ -251,6 +251,24 @@ requires_any_configs_disabled() {
     fi
 }
 
+TLS1_2_KEY_EXCHANGES_WITH_CERT="MBEDTLS_KEY_EXCHANGE_DHE_RSA_ENABLED \
+                                MBEDTLS_KEY_EXCHANGE_DHE_RSA_ENABLED \
```

Comment:
```suggestion
```

Reply:
In fact one of the two should have been `MBEDTLS_KEY_EXCHANGE_RSA_ENABLED`.